Handler/Sites.hs

```diff
@@ -85,17 +85,25 @@ embedExample msiteId = do
 
 siteForm :: Maybe Site -> Form Site
 siteForm msite = renderBootstrap3 BootstrapBasicForm $ Site
-    <$> areq textField (bfl "Name") (siteName <$> msite)
-    <*> areq textField (bfl "Base URL") (siteBaseUrl <$> msite)
-    <*> areq textField (bfl "RSS Author") (siteRssAuthor <$> msite)
-    <*> areq textField (bfl "RSS Title") (siteRssTitle <$> msite)
-    <*> areq htmlField (bfl "RSS Description") (siteRssDescription <$> msite)
-    <*> areq textField (bfl "RSS Language") (siteRssLanguage <$> msite)
+    <$> areq textField (fs "Name" "my-site") (siteName <$> msite)
+    <*> areq textField (fs "Base URL" "https://example.com") (siteBaseUrl <$> msite)
+    <*> areq textField (fs "Comments Feed Author" "My Site") (siteRssAuthor <$> msite)
+    <*> areq textField (fs "Comments Feed Title" "My Site Comments") (siteRssTitle <$> msite)
```

Comment:
It still seems like it would be an improvement to standardize/generate these two fields. I'm not sure why somebody would want to customize the description for their comments feed.